tests/integration/__snapshots__/integration.test.ts.snap

```diff
@@ -2,150 +2,133 @@
 
 exports[`outputs accessibility violation information for the demo app 1`] = `
 "$ axe-storybook
+[chromium] accessibility
+  button
+    ✓ Button with text (should pass)
+    ✓ Button with emoji and label (should pass)
+    1) Button with no discernible text (should fail)
+    2) Button with insufficient color contrast (should fail)
+    - Button with insufficient color contrast but skipped (should pass)
+    3) Multiple buttons with insufficient color contrast (should fail)
+  input
+    ✓ Input with label (should pass)
+    4) Input without label (should fail)
+    - Input without label but skipped (should pass)
+    5) Input without label and invalid role (should fail)
+    ✓ Input without label but \\"label\\" rule is disabled (should pass)
 
+4 passing
+5 failing
+2 pending
 
-  [chromium] accessibility
-    button
-      ✓ Button with text (should pass) 
-      ✓ Button with emoji and label (should pass) 
-      1) Button with no discernible text (should fail)
-      2) Button with insufficient color contrast (should fail)
-      - Button with insufficient color contrast but skipped (should pass)
-      3) Multiple buttons with insufficient color contrast (should fail)
-    input
-      ✓ Input with label (should pass) 
-      4) Input without label (should fail)
-      - Input without label but skipped (should pass)
-      5) Input without label and invalid role (should fail)
-      ✓ Input without label but \\"label\\" rule is disabled (should pass) 
+1) [chromium] accessibility
+     button
+       Button with no discernible text (should fail)
 
+       Detected the following accessibility violations!
 
-  4 passing 
-  2 pending
-  5 failing
+       1. button-name (Buttons must have discernible text)
 
-  1) [chromium] accessibility
-       button
-         Button with no discernible text (should fail):
-     AssertionError [ERR_ASSERTION]: 
```

Comment:
nice that custom formatting makes these a bit better 🤓

Reply:
Yeah, it was a little weird before.